src/twisted/internet/base.py

```diff
@@ -785,7 +787,9 @@ def run(self):
 
     seconds = staticmethod(runtimeSeconds)
 
-    def callLater(self, _seconds, _f, *args, **kw):
+    # We need to ignore mypy errors here because the '_f' parameter is
+    # not in IReactorTime.callLater
+    def callLater(self, _seconds, _f, *args, **kw):  # type: ignore[override]
```

Comment:
just use the signature from the interface.

Reply:
How?  The signature from the interface takes 4 args, but this function signature takes 5 args.

Reply:
I've moved this to https://github.com/twisted/twisted/pull/1325